.github/dependabot.yml

```diff
@@ -16,6 +16,7 @@ updates:
   labels:
     - "ok-to-test"
     - "kind/cleanup"
+    - "release-note-none"
```

Comment:
Will adding this label engage the k8sbot (which is normally done via comment `/release-note-none`)?

Reply:
yes, behavior should be the same as "ok-to-test" (prow just looks for that label)